lib/multipart.js

```
@@ -424,8 +425,7 @@ Parser.handle = function(req, res, next, options) {
 
   parser.on('error', function(err) {
     if (err.hasCustomHandler !== true) {
-      req.destroy();
-      next(err);
+      createError(err.message,  options.handleGenericError)
```

What happens if there is no generic handler provided either?

---

Sorry, I'm confused so I'm likely seeing past something.

Isn't this the on error handler? It looks like it always calls `createError()` which will just ignore the error if there is no `options.handleGenericError`?

---

yes, but for some of the errors (like no boundary) options didn't exist, so it never went to data, so I switched it back to using the generic

---

Yes, no issue with there being an option for a generic handler function.
However, the option is optional so we still need to cater for the ultimate fallback case where no generic handler function is provided in the options.